### PR TITLE
add yarn cache & yarn js testing

### PR DESCRIPTION
**Pull request by OmgImAlexis**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by fernandog** [comment]
@OmgImAlexis i flake the dreed_hook file. Feel free to update the docstring to explain better those functions

File: dredd_hook.py [hunk 1]
```diff
@@ -1,12 +1,16 @@
+"""Dredd hoook."""
```
Comment (OmgImAlexis) [inline dredd_hook.py]: Double o not triple.

**Comment by OmgImAlexis** [comment]
That's fine as is. I didn't even both with adding docstrings because of the simplicity of the whole thing.

File: dredd_hook.py (hunk 1 above)
Comment (fernandog) [inline dredd_hook.py]: its correct this way

Comment (OmgImAlexis) [inline dredd_hook.py]: What? Hook is spelt hook not hoook?

Comment (fernandog) [inline dredd_hook.py]: ahhh i got it now lol

**Comment by fernandog** [comment]
travis:

$ xo 
  static/js/manage/backlog-overview.js:70:18
  ⚠  70:18  Expected error to be handled.   handle-callback-err
  ⚠  83:18  Expected error to be handled.   handle-callback-err
  ✖  70:27  err is defined but never used.  no-unused-vars
  ✖  78:17  Extra space after key status.   key-spacing
  ✖  83:27  err is defined but never used.  no-unused-vars
  2 warnings
  3 errors
error Command failed with exit code 1.

**Comment by fernandog** [comment]
@OmgImAlexis all good. approved.  anything else?

**Pull request merged**